### PR TITLE
feat: add supportActiveProfiles field & extConfig support add activeP…

### PR DESCRIPTION
**Pull request by fudali113**
https://github.com/spring-cloud-incubator/spring-cloud-alibaba/issues/297

### CODE REVIEW

**Comment by fangjian0423** [comment]
Thank you for your pull request. 

We will review it soon.

**Comment by flystar32** [comment]
@pbting  help to take a look at this pull request

File: spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/client/NacosPropertySourceLocator.java [hunk 1]
```diff
@@ -129,9 +129,11 @@ private void loadExtConfiguration(CompositePropertySource compositePropertySourc
 
 		for (NacosConfigProperties.Config config : extConfigs) {
 			String dataId = config.getDataId();
-			String fileExtension = dataId.substring(dataId.lastIndexOf(".") + 1);
-			loadNacosDataIfPresent(compositePropertySource, dataId, config.getGroup(),
-					fileExtension, config.isRefresh());
+			String fileExtension = dataId.substring(dataId.lastIndexOf(DOT) + 1);
+			String dataIdPrefix = dataId.substring(0, dataId.lastIndexOf(DOT));
+
+			loadDataIdByActiveProfilesConfiguration(compositePropertySource, config.getGroup(),
```
[the remark this answers is not in the document]
Comment (flystar32) [reply spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/client/NacosPropertySourceLocator.java]: in addition, we think it's dangerous to auto refresh shared config, so you can't simply set it's auto refresh to true.

**Comment by flystar32** [comment]
Sorry for review this pull request so late due to a non-technical factors, Thank you for your contribution to Spring Cloud Alibaba!

**Pull request closed without merging**